_overviews/scala3-book/fp-pure-functions.md

```diff
@@ -98,6 +98,21 @@ def double(i: Int): Int = i * 2
 
 {% endtabs %}
 
+The next example is bit more tricky. Here, `i` is not passed as a parameter, but instead referenced directly from the outside.
+This works in Scala because functions act as closures - they can capture the state around them. As long as that state is *immutable*, such a closure is still considered pure.
```

Comment:
Not sure this is the best wording since we usually use "state" to talk about mutable state. Not all of the time. I would suggest "capture values from enclosing scopes".